packages/suite/src/hooks/wallet/useBitcoinAmountUnit.ts

```diff
@@ -5,6 +5,13 @@ import { useActions } from '@suite-hooks/useActions';
 import * as walletSettingsActions from '@settings-actions/walletSettingsActions';
 import { hasNetworkFeatures } from '@wallet-utils/accountUtils';
 
+export const BITCOIN_UNIT_ABBREVIATION = {
+    [PROTO.AmountUnit.BITCOIN]: 'BTC',
+    [PROTO.AmountUnit.MICROBITCOIN]: 'μBTC',
+    [PROTO.AmountUnit.MILLIBITCOIN]: 'mBTC',
```

Comment:
Does it make sense to have micro and milli? They are not mentioned anywhere in the code, I'd not include those.

Reply:
They are mentioned in `transport` => `PROTO.AmountUnit`. I will delete it if we will never extend it.

Reply:
Moreover, I need to use `// @ts-ignore` to be able to translate `state.wallet.settings.bitcoinAmountUnit` into abbreviation

Reply:
In general, I would prefer to not have anything unused in the code, but AmountUnit is defined also here https://github.com/trezor/trezor-common/blob/master/protob/messages-bitcoin.proto#L49 so I believe we could keep it here.

Reply:
@matejkriz yes, It is [defined in firmware some time ago](https://github.com/trezor/trezor-firmware/pull/1369), before connect and before I started to work on units for sute. However the final issue for sute, the one that is attatched to this PR is only for sats and BTC. Wether to keep the other 2 on the firmware side is another question but I don't think that they should be mentioned on suite's side at all to avoid misunderstanding, since micro and milli are mentioned only here in the whole suite now and it makes little sense :(

Reply:
we could remove them from here but then we should redefine the suite types everywhere instead of `PROTO.AmountUnit` use `PROTO.AmountUnit.BITCOIN | PROTO.AmountUnit.SATOSHI`